### ISSUE TITLE
Dropdown menu in custom cell doesn’t work

### ISSUE THREAD

**Issue by bggolding** [jbetancur/react-data-table-component#615]
## Issue Check list
- [x] Agree to the [Code of Conduct](https://github.com/jbetancur/react-data-table-component/blob/master/CODE-OF-CONDUCT.md)
- [x] Read the README
- [x] You are using React 16.8.0+
- [x] You installed `styled-components`
- [x] Include relevant code or preferably a [code sandbox](https://codesandbox.io/embed/react-data-table-sandbox-ccyuu
)

## Describe the bug
In the table I am using, I have a custom cell which contains a dropdown list of actions that I want to be able to use for that row. The dropdown list works outside the table and also in a table using react-bootstrap-table-next but not react-data-table-component.

## To Reproduce
1. Open code sandbox (https://codesandbox.io/live/MjgEwm)
2. Click on “Actions” in table
3. Note that it does not open
4. Click on “Actions” at top of page (rendered outside table), notice that the dropdown works.

## Expected behavior
I expect the dropdown to open when rendered inside the table.

## Code Sandbox, Screenshots, or Relevant Code
Here is a sandbox that demonstrates the dropdown working outside the table and not responding when rendered inside the table:

https://codesandbox.io/live/MjgEwm

## Versions (please complete the following information)
 - React 16.13.1
 - Styled Components 5.1.1
 - OS: MacOS
 - Browser Chrome and Safari

**Comment by jbetancur** [comment]
the sandbox is no longer available when I click it

**Comment by bggolding** [comment]
I’ve recreated it here https://codesandbox.io/s/react-data-table-sandbox-9ev0g

**Comment by jbetancur** [comment]
Thank you. So yes this is def a bug. It has to do with https://github.com/jbetancur/react-data-table-component/blob/b9e6b888beb5c49b7655f7c6e637fb3ea858a327/src/DataTable/TableCell.js#L11

where RDT is applying styles to the first div element in a TableCell.
What should happen should be ignored when the element is column.button: true

I should be able to fix this pretty quickly in a 6.9.4

**Comment by jbetancur** [comment]
@bggolding This should now be fixed. Make sure that you set `column.button: true` which is intended for when you add buttons to a cell (but should now work for your case)

```js
  {
    name: "Actions",
    button: true,
    cell: row => <ListActions row={row} />
  }
```

**Comment by jbetancur** [comment]
Sorry update on this again @bggolding 

Anytime you use `cell` the overflow will no longer be applied. The reason for overflow: hidden is so default cells can wrap/ellipses

However, you still want to use `button: true` as this enables additional css and disabled row click even behaviors.

The fix for this is in 6.9.5 which I am releasing shortly

**Comment by bggolding** [comment]
Thanks.
